src/css/imports/controlbar.less

```diff
@@ -147,3 +148,14 @@
     }
   }
 }
+
+.jw-flag-ads-vpaid,
+.jw-flag-autostart,
+.jw-flag-user-inactive.jw-state-playing,
+.jw-flag-user-inactive.jw-state-buffering {
+  &:not(.jw-flag-media-audio):not(.jw-flag-audio-player):not(.jw-flag-ads-vpaid-controls):not(.jw-flag-casting) {
+    .jw-controlbar {
+      display: none;
```

Comment:
Since controlbar is a child of controls which is hidden I've updated jw-flag-controls-hidden to not change the controlbar display.